From: Backport of https://github.com/Textualize/pytest-textual-snapshot/pull/19
Subject: [PATCH] Fix package homepage url in pyproject.toml

diff --git a/pyproject.toml b/pyproject.toml
index 3e3ff40..4fb9b73 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -6,7 +6,7 @@ authors = ["Darren Burns <darren@textualize.io>"]
 maintainers = ["Will McGugan <will@textualize.io>"]
 license = "MIT"
 readme = "README.md"
-homepage = "https://github.com/darrenburns/pytest-textual-snapshot"
+homepage = "https://github.com/Textualize/pytest-textual-snapshot"
 classifiers = [
     "Development Status :: 5 - Production/Stable",
     "Framework :: Pytest",
